lms/static/certificates/sass/_views.scss

```diff
@@ -11,3 +11,35 @@
 // #DISTINCTION CERT
 // ------------------------------
 
+
+// ------------------------------
+// #INVALID
+// ------------------------------
+
+.wrapper-content.status-invalid {
```

Comment:
Stinks that we can't use a body element-based class here, but if we can't this is the second best selector, I suppose. Just noting more for history than anything.